native/src/components/NavigationTiles.tsx

```diff
@@ -52,16 +53,15 @@ const NavigationTiles = ({ tiles, theme }: NavigationTilesProps): ReactElement =
     setPercentageScrolled(contentOffset.x / (contentSize.width - layoutMeasurement.width))
   }
 
+  const scrolledToStart = percentageScrolled === 0
+  const scrolledToEnd = percentageScrolled >= scrolledToEndThreshold
+
+  const scrollToStart = () => scrollViewRef.current?.scrollTo({ x: 0, y: 0, animated: true })
+  const scrollToEnd = () => scrollViewRef.current?.scrollToEnd({ animated: true })
```

Comment:
I guess this is fine as long as we don't get more icons there. Which I think is not planned?

Reply:
Agreed and yes, don't think more icons are planned :+1: